config/src/main/resources/schema.json

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
Thinking more about this, it would probably be easier to use an existing dependency graph library and format to store and parse this info - here's one example:
https://netflix.github.io/falcor/documentation/jsongraph.html

Reply:
Very interesting, reading this documentation and will suggest an implementation using JsonGraph. Thank you.